jobs/table_connection_la_1

```diff
@@ -0,0 +1,47 @@
+---
+title: Backend/Web Developer for Grassroots Start-Up
+company: Table Connection
+url: https://github.com/lawloretienne
+location: Los Angeles
```

Comment:
Consider specifiying the location in a more precise way, e.g. Downtown, New York City, USA